src/app/retrospective/retrospect-task-modal/retrospect-task-modal.component.ts

```diff
@@ -255,7 +282,9 @@ export class RetrospectTaskModalComponent implements OnDestroy {
                 valueFormatter: (cellParams) => {
                     return MEMBER_TASK_ROLES_LABEL[cellParams.value];
                 },
-                editable: isSprintEditable,
+                editable: (params: IsColumnFuncParams) => {
```

Comment:
Either specify the type of params for all columns or don't specify the type at all to keep things consistent.

Reply:
we should specify the type as long as there are no other concerns

Reply:
Added elsewhere, except for GridReadyEvent for which seems to be a type mismatch.